commands/template_store.go

```diff
@@ -0,0 +1,21 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	templateCmd.AddCommand(templateStoreCmd)
+}
+
+var templateStoreCmd = &cobra.Command{
+	Use:   `store [COMMAND]`,
+	Short: `Command for pulling and listing templates from store`,
+	Long: `This command provides the list of the templates from the store
+	 that we support and pull them by name`,
+	Example: `  faas-cli template store list --verbose
+  faas-cli template store ls -v
```

Comment:
Please remove the source flag. This was not part of the design. If anything the part we need has been missed which is to provide a custom URL for a template store.

Reply:
source removed ✔️